README.md

```diff
@@ -79,24 +79,34 @@ Rehabify is a comprehensive management platform designed for rehabilitation and
     npm start
     ```
 
+---
+
 ## Usage
 
 1. **Access the platform**:
-    - Open your browser and navigate to `http://localhost:5173` for the frontend.
-    - The backend server runs on `http://localhost:3000`.
+    - Frontend: Navigate to `http://localhost:5173`.
+    - Backend: The server runs on `http://localhost:3000`.
 
-2. **Login as Super Admin**:
-    - Use the provided super admin credentials to log in and manage the platform.
+2. **Login**:
+    - Use the super admin credentials to manage the platform.
+  
+3. **Search for a Nasha Mukti Kendra**:
+    - Use the search feature to locate nearby centers.
 
-3. **Search for Nasha Mukti Kendra**:
-    - Use the search functionality to find the nearest centers.
+---
 
 ## Contributing
 
-We welcome contributions! Please fork the repository and submit pull requests.
+We welcome contributions! Please follow these guidelines:
 
-## License
+- **Branching**: Ensure you raise a pull request to the `develop` branch.
+- **Forking**: Fork the repository, make your changes, and submit a pull request.
+- **Code Reviews**: All submissions will undergo code review.
 
-This project is licensed under the MIT License.
+For any issues or suggestions, feel free to create an issue or contact us directly.
 
+---
 
+## License
```

Comment:
also add the contribution images at the end of readme 

![{399AD2B6-F0E4-4A88-B980-BD1FA763785F}](https://github.com/user-attachments/assets/a9aaa9e6-d5b0-47fe-aadf-901e0687dd3c)

like this 
https://github.com/ratnesh-maurya/Educational-Tech-Exploration

### also you can take reference from this  project readme too and add more elements 

- https://github.com/Kushal997-das/Project-Guidance